automatminer/automl/adaptors.py

```diff
@@ -118,9 +118,9 @@ def fit(self, df, target, **fit_kwargs):
         self._features = df.drop(columns=target).columns.tolist()
```

Comment:
Again, not sure why this adaptor stuff is being changed....

Reply:
You're right, the logger attribute should not be changed. I thought the presence of the underscore was the reason I was getting issues in the other parts of that file, but it must have been something else. I'll change it back